Automation/wdio/homeworks/hw_1/test/specs/test.e2e.ts

```diff
@@ -0,0 +1,66 @@
+enum ErrorMessages {
+    SUCCESS = 'Successfully registered! Please, click Back to return on login page',
+    USERNAME_REQUIRED = 'Username is required',
+    PASSWORD_RQUIRED = 'Password is required',
+    USERNAME_MIN_LENGTH = 'Username should contain at least 3 characters',
+    USERNAME_SPACES = 'Prefix and postfix spaces are not allowed is username',
+    INVALID_DATA = 'Please, provide valid data',
+    PASSWORD_MIN_LENGTH = 'Password should contain at least 8 characters',
+    PASSWORD_UPPER = 'Password should contain at least one character in upper case',
+    PASSWORD_LOWER= "Password should contain at least one character in lower case"
+}
+
+
+const testData = [    
```

Comment:
Separate test data like `valid` | `notValid`
Add second test `should not login...`